dozer-sql/Cargo.toml

```diff
@@ -19,6 +19,7 @@ enum_dispatch = "0.3.12"
 linked-hash-map = { version = "0.5.6", features = ["serde_impl"] }
 metrics = "0.21.0"
 multimap = "0.9.0"
+wasmtime = { version = "9.0.4", optional = true }
```

Comment:
We don't need this dependency?

Reply:
This is the wasm runtime that we use. This runs the actual wasm functions